Created pattern Features with Images

kafleg opened the pull request:
This pattern is copied from home.php pattern and created single part of it

kafleg committed:
Created pattern Features with Images

This pattern is copied from home.php pattern and created single part of it

maneshtimilsina commented:
@kafleg I request you to check the following issues I found on this pattern.

1. Translation is missing.
2. End your code/file with an empty new line.
3. Use `get_template_directory_uri` instead of `get_stylesheet_directory_uri` to load assets.

kafleg commented:
> @kafleg I request you to check the following issues I found on this pattern.
> 
> 1. Translation is missing.
> 2. End your code/file with an empty new line.
> 3. Use `get_template_directory_uri` instead of `get_stylesheet_directory_uri` to load assets.

Thank you for the suggestion. I'm going to work on it.

kafleg committed:
Change get_stylesheet to get_template_directory

As discussed in the issues #101 we need to change the get_stylesheet_directory_uri() to get_template_directory_uri()

kafleg committed:
fixed automatic check

kafleg committed:
Translation added

kafleg committed:
Fixed space

kafleg committed:
fixing Code Quality / Lint: PHP (pull_request)

carolinan commented on patterns/features-with-images.php:
```diff
@@ -0,0 +1,99 @@
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"textAlign":"center"} -->
+<h2 class="wp-block-heading has-text-align-center"><?php echo esc_html_x('An array of resources', 'Sample content for heading of the section.', 'twentytwentyfour'); ?></h2>
```
```suggestion
<h2 class="wp-block-heading has-text-align-center"><?php echo esc_html_x('An array of resources', 'Sample content for heading of the section.', 'twentytwentyfour' ); ?></h2>
```

carolinan committed:
Update patterns/features-with-images.php

MaggieCabrera commented on patterns/features-with-images.php:
```diff
@@ -0,0 +1,99 @@
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6vh","bottom":"8vh"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:6vh;padding-bottom:8vh"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
```
this needs to be wide aligned

MaggieCabrera commented on patterns/features-with-images.php:
```diff
@@ -0,0 +1,99 @@
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:paragraph -->
+<p><strong>✓</strong> <?php echo esc_html__('Collaborate with fellow architects', 'twentytwentyfour'); ?></p>
```
I think this will be an issue for translators of RTL languages. I'm not an expert here but I would include the symbol into the translation string (using sprintf since we have the strong tag there) and add a comment for translators suggesting this is a list of items. I'd love other's people input on this, since I see how this could be approached in multiple ways, but probably this is the simplest right now.

MaggieCabrera replied:
Actually, disregard that, check the conversation about the checkmarks going on at https://github.com/WordPress/twentytwentyfour/pull/150, and lets follow that lead instead.

MaggieCabrera commented:
If we are making a pattern that is being extracted from the one in home, we should delete the duplicated code there and include the pattern block that links to the new one.

kafleg committed:
Fixed align wide

kafleg commented:
> If we are making a pattern that is being extracted from the one in home, we should delete the duplicated code there and include the pattern block that links to the new one.

Sure, but I was waiting to merge this PR to replace it from home. (I can work in it. )

MaggieCabrera commented:
> Sure, but I was waiting to merge this PR to replace it from home. (I can work in it. )

I think it's better if both things happen in the same PR

maneshtimilsina commented:
> I think it's better if both things happen in the same PR

I think it will be better to replace `home.html` after all patterns get approved to avoid merge conflict.

MaggieCabrera committed:
changed spacings to presets

MaggieCabrera committed:
linting
